### PR TITLE
docs: add TUI_IS_MOBILE_RES to tokens page

### LINKED ISSUE #2813: 📚 - Token TUI_IS_MOBILE_RES disappeared in v2.64 docs
### What is the affected URL?

https://taiga-ui.dev/v2/utils/tokens

### Description

TUI_IS_MOBILE_TOKEN was in previous versions v2, but in current version doesn't

Please, add this token in v2 doc.

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by ddubrava**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

Closes #2813

## What is the new behavior?

The missing token (TUI_IS_MOBILE_RES) is added to the documentation page.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/utils/tokens/examples/7/index.html [hunk 1]
```diff
@@ -1 +1,4 @@
-<p i18n>Provide an array of 12 strings for localized names for months to be used in TuiMonthPipe</p>
+<p i18n>
+    A token with a factory. It compares the viewport width and the value from the TUI_MEDIA token to detect users with
+    mobile resolution
+</p>
```
Comment (waterplea) [inline projects/demo/src/modules/utils/tokens/examples/7/index.html:4]: Let's add an `async` pipe with that stream so people could see it in action.

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/utils/tokens/examples/7/index.html (hunk 1 above)
Comment (ddubrava) [reply projects/demo/src/modules/utils/tokens/examples/7/index.html:4]: this token also lives in v3. so should we raise another issue for this change?

Comment (waterplea) [reply projects/demo/src/modules/utils/tokens/examples/7/index.html:4]: No need for a separate issue, but if you can make a quick PR, that would be great :)

Comment (ddubrava) [reply projects/demo/src/modules/utils/tokens/examples/7/index.html]: done. if it works for you let me know. I'll create the same PR for the current version

**Pull request merged**